src/global/types.h

```diff
@@ -1741,6 +1756,9 @@ typedef struct CAMERA_INFO {
     /* 0058 */ ITEM_INFO *last_item;
     /* 005C */ OBJECT_VECTOR *fixed;
     /* 0060 end */
+    // used for the manual camera control
+    int16_t additional_angle;
```

Comment:
As rr- said, different file that needs the change from "additional_angle" to  "angle_override"